src/components/Header/Menu/constants/pathToParent.ts

```diff
@@ -0,0 +1,3 @@
+export enum pathToParent {
```

Comment:
you need to create variable with all route paths

Reply:
fixed